external_deps/build.sh

```diff
@@ -759,6 +772,19 @@ setup_macos-amd64-default() {
 	export NASM="${PWD}/${BUILD_BASEDIR}/prefix/bin/nasm" # A newer version of nasm is required for 64-bit
 }
 
+# Set up environment for 32-bit i686 Linux
+setup_linux-i686-default() {
+	HOST=i386-unknown-linux-gnu
+	CROSS=
+	MSVC_SHARED=(--disable-shared --enable-static)
```

Comment:
Why are these flags required?

Reply:
MSVC_SHARED is just goofily named. It defines whether static or shared libraries are preferred for the platform. MSVC is the only one where shared is preferred.